app/api/orders.py

```diff
@@ -437,18 +437,6 @@ def after_update_object(self, order, data, view_kwargs):
                 send_notif_ticket_purchase_organizer(order.event.owner, order.invoice_number, order_url,
                                                      order.event.name, order.identifier)
 
-    def before_delete_object(self, order, view_kwargs):
-        """
-        method to check for proper permissions for deleting
-        :param order:
-        :param view_kwargs:
-        :return:
-        """
-        if not has_access('is_coorganizer', event_id=order.event.id):
-            raise ForbiddenException({'source': ''}, 'Access Forbidden')
-        elif order.amount and order.amount > 0 and (order.status == 'completed' or order.status == 'placed'):
-            raise ConflictException({'source': ''}, 'You cannot delete a placed/completed paid order.')
```

Comment:
Why delete this?

Reply:
@iamareebjamal This code never runs. Reference: https://github.com/fossasia/open-event-server/issues/6450